### PR TITLE
[Computer Vision] Create CV API v3.0-preview

### PR DESCRIPTION
**Pull request by TFR258**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by TFR258** [comment]
@bojunehsu , the PR is ready

**Comment by TFR258** [comment]
We are currently waiting on the API review

**Comment by TFR258** [comment]
@ryogok , This is our latest APIM set of changes for the OCR release

**Review by anuchandy — APPROVED** [review]
Compared this v3 with current v2, changes looks minimal. LGTM.

**Comment by anuchandy** [comment]
@TFR258 did we deploy this version of vision to at-least one production region? a deployment is needed for PR to merge to master.

**Comment by anuchandy** [comment]
@TFR258 is this deployed to at least one production region?

**Comment by TFR258** [comment]
Not yet, But we are getting very close.
I’ll ping this thread when we do =D

Thank you for following up.

Tom

**Comment by TFR258** [comment]
Hi @anuchandy ,
We have rolled out in Canary and one prod regions.
I see you have marked the PR as DO NOT MERGE.
Let me know when it would be best for us to sync.
Thanks,
Tom

**Comment by TFR258** [comment]
@anuchandy , can you help us approve this PR?

File: specification/cognitiveservices/data-plane/ComputerVision/preview/v3.0/Ocr.json [hunk 1]
```diff
@@ -0,0 +1,447 @@
...
+    "ReadOperationResult": {
+      "type": "object",
+      "description": "OCR result of the read operation.",
+      "properties": {
+        "status": {
+          "description": "Status of the read operation.",
+          "$ref": "#/definitions/OperationStatus"
+        },
+        "createdDateTime": {
+          "type": "string",
```
Comment (lmazuel) [inline specification/cognitiveservices/data-plane/ComputerVision/preview/v3.0/Ocr.json]: You could also add "format" if your date format is consistent (always ISO8601) for instance:
- `format:date-time` means ISO8601 (full date + hours)
- `format:date` means ISO8601 (full date + hours)
- `format:date-time-rfc1123` means RFC1123

File: specification/cognitiveservices/data-plane/ComputerVision/preview/v3.0/Ocr.json [hunk 2]
```diff
@@ -0,0 +1,447 @@
...
+          "description": "Get UTC date time the batch operation was submitted.",
+          "x-nullable": false
+        },
+        "lastUpdatedDateTime": {
+          "type": "string",
```
Comment (lmazuel) [inline specification/cognitiveservices/data-plane/ComputerVision/preview/v3.0/Ocr.json]: See comment above

**Review by lmazuel — APPROVED** [review]

**Pull request merged**